src/openrct2/network/Network.cpp

```diff
@@ -1588,7 +1595,7 @@ void Network::Server_Send_MAP(NetworkConnection* connection)
         }
         return;
     }
-    size_t chunksize = CHUNK_SIZE;
+    size_t chunksize = 128; //CHUNK_SIZE;
```

Comment:
Seems to be a mistake? I'd expect it to introduce another constant, if `CHUNK_SIZE != 128` or use `CHUNK_SIZE` directly.

Reply:
I could swear I removed this before commiting, good catch.